.env

```diff
@@ -21,6 +21,10 @@ REACT_APP_TERMS_OF_SERVICES=
 
 # Can be one of onyxia, france or ultraviolet
 REACT_APP_THEME_ID=onyxia
+# Overrides fields of the color palette, needs to be single line JSON corresponding to the fields in paletteOverride
+# If this is set, then REACT_APP_THEME_ID is ignored
+# Example: '{"focus": {"main": "#00824D","light": "#00824D","light2": "#1A9D49"}}'
```

Comment:
```suggestion
# This parameter allows you to override some, or all, of the color palette values for the selected theme.
#
# Please adhere to the JSON format outlined in the following object:
# https://github.com/InseeFrLab/onyxia-ui/blob/0fa8667227aac328cb685cc140f3cd2186956304/src/lib/color.ts#L15-L53
#
# Should you desire to personalize the color palette without the assistance of a UI designer, 
# there is a simple way. You can copy and paste the default palette and request ChatGPT to 
# propose a variation. For instance, you might want to emphasize a specific color, like 
# "TypeScript's blue". ChatGPT can use that as a reference to generate a new color scheme 
# tailored to your preference.
#
# Example:  
#
# PALETTE_OVERRIDE: |
#   { 
#     "focus": { 
#         "main": "#00824D", 
#          "light": "#00824D"
#      } 
#   }
```

The JSON must be formatted as a single line in the .env file, but we anticipate that users will define these variables from the values.json of their Helm charts. That's why we provide examples in this particular format.